SwiftCheck/Operators.swift

```diff
@@ -54,6 +54,23 @@ public func <?>(p : Testable, s : String) -> Property {
 	return label(s)(p: p)
 }
 
+infix operator ^&^ {
+	associativity right
+	precedence 110
+}
+
+/// Takes the nondeterministic conjunction of two properties and treats them as a single large
+/// property.
+///
+/// The resulting property makes 100 random choices to test either the left or right property.  Thus,
+/// running multiple test cases will result in distinct arbitrary sequences of each property being
+/// tested.
+public func ^&^(p1 : Testable, p2 : Testable) -> Property {
```

Comment:
This is QuickCheck's notation.  I couldn't find very many other historical examples of non-deterministic choice operators that looked good or weren't just `amb` hidden behind AoN.